contracts/BRDLockup.sol

```diff
@@ -0,0 +1,136 @@
+pragma solidity ^0.4.15;
+
+import "zeppelin-solidity/contracts/ownership/Ownable.sol";
+import "zeppelin-solidity/contracts/math/SafeMath.sol";
+
+
+/**
+ * Contract BRDLockup keeps track of a vesting schedule for pre-sold tokens.
+ * Pre-sold tokens are rewarded up to `numIntervals` times separated by an
+ * `interval` of time. An equal amount of tokens (`allocation` divided by `numIntervals`)
+ * is marked for reward each `interval`.
+ *
+ * The owner of the contract will call processInterval() which will
+ * update the allocation state. The owner of the contract should then
+ * read the allocation data and reward the beneficiaries.
+ */
+contract BRDLockup is Ownable {
```

Comment:
I think this would be sufficient for the crowdsale token but better to have multiple owners and some kind of voting mechanism among owners to prevent losing all controls just by losing single private key.